modules/proc_splunk-config.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: PROCEDURE
+[id="proc_splunk-config"]
+= Configuring {productname} to use Splunk
+
+Use the following procedure to configure {productname} to use Splunk.
+
+.Prerequisites
+
+* You have installed Splunk and created a username.
+* You have generated a Splunk bearer token.
+
+.Procedure
+
+* Open your {productname} `config.yaml` file and add the following configuration fields:
++
+[source,yaml]
+----
+---
+LOGS_MODEL: splunk
+LOGS_MODEL_CONFIG:
+  	producer: splunk
+  	splunk_config:
+    		host: http://<user_name>.remote.csb <1>
+    		port: 8089 <2>
+    		bearer_token: <bearer_token> <3>
+			url_scheme: <http/https> <4>
+    		verify_ssl: False <5>
+    		index_prefix: <splunk_log_index_name> <6>
+---
+----
+<1> String. The Splunk cluster endpoint.
+<2> Integer. The Splunk management cluster endpoint port.
```

Comment:
should we call out that management port is different from the splunk gui hosted port? or we could specify where in UI to find the management port. wdyt?

Reply:
Done!